src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -137,6 +139,9 @@ def _merge_lists(base, patches):
     def tearDown(self):
         for processor in self._processors_to_reset:
             processor.reset()
+        if self.random_config_dir:
+            import shutil
+            shutil.rmtree(self.cli_ctx.config.config_dir, ignore_errors=True)
```

Comment:
Well... yes. `rmtree` can fail intermittently on Windows, consider using 

https://github.com/Azure/azure-cli/blob/57577782547064257d69da916683a46e7c25e1af/src/azure-cli-core/azure/cli/core/util.py#L1317

Reply:
Oh, yes. I forget that Windows bug.